src/common/schemas/GaugeViewProperties.yml

```diff
@@ -42,9 +42,5 @@
       type: string
     tickSuffix:
       type: string
-    staticLegend:
-      $ref: "./StaticLegend.yml"
```

Comment:
assuming this removal of static legend was intentional

Reply:
Yes this is intentional. Gauge will not have a static legend.